examples/v1alpha3/file-metricscollector-example.yaml

```diff
@@ -55,7 +55,7 @@ spec:
                   imagePullPolicy: Always
```

Comment:
Can you change image tag from 1.0 to `docker.io/kubeflowkatib/pytorch-mnist` in v1alpha3 and v1beta1.

Reply:
Done